cypress/e2e/navigation.cy.js

```diff
@@ -0,0 +1,6 @@
+describe('Navigation on the site of clothes', () => {
+  it('Access the page and verify the address', () => {
+    cy.visit('/');
+    cy.url().should('eq', 'http://automationpractice.com/index.php');
```

Comment:
Reflector this command using a parametrization approach like the following:
cy.url().should('eq', Cypress.env('baseUrl'));
Read this one: [Cypress.env](https://docs.cypress.io/api/cypress-api/env)